tests/syndication_tests/tests.py

```diff
@@ -47,8 +47,8 @@ def assertChildNodes(self, elem, expected):
 
     def assertChildNodeContent(self, elem, expected):
         for k, v in expected.items():
-            self.assertEqual(
-                elem.getElementsByTagName(k)[0].firstChild.wholeText, v)
+            with self.subTest(tagname=k, value=v):
```

Comment:
I don't think it's useful to include `value=v` in `subTest()` since that would appear in the failing assertion. (Similarly, for `output` in the other tests.)